lua/nvim-treesitter/textobjects/attach.lua

```diff
@@ -10,18 +10,18 @@ function M.make_attach(normal_mode_functions, submodule)
     lang = lang or parsers.get_buf_lang(bufnr)
 
     for _, function_call in pairs(normal_mode_functions) do
-      for mapping, query in pairs(config[function_call] or {}) do
+      for mapping, config_queries in pairs(config[function_call] or {}) do
         if not queries.get_query(lang, "textobjects") then
-          query = nil
+          config_queries = nil
         end
-        if query then
```

Comment:
This looks very nice! Would it be possible to leave the old config working? E.g. by only appling the table.concat if it is a table `type(config_queries) == 'string' and config_queries or table.concat(config_queries)`

Reply:
This is a good idea; I've implemented it in the `move` and `swap` functions themselves rather than at all of the call-sites.

Reply:
But how will `table.concat` work then?

Reply:
Oops, sorry. I pushed another commit to fix it, which also fixed LSP interop since it uses the same `make_attach` function as `move` and `swap`.